app/code/core/Mage/Adminhtml/Block/Template.php

```diff
@@ -50,10 +50,7 @@ public function getFormKey()
      */
     public function isOutputEnabled($moduleName = null)
     {
-        if ($moduleName === null) {
-            $moduleName = $this->getModuleName();
-        }
```

Comment:
Removal of these lines technically changes the behavior. If `$moduleName = null`, before we would check `Mage_Adminhtml` and now we would check `Mage_Core`.

However, I only see two calls to this method, and in both cases `$this->isOutputEnabled('Mage_AdminNotification')`